examples/skip_plot_field_lines.py

```diff
@@ -0,0 +1,52 @@
+"""
+================================
+Plotting Field Lines from Pfsspy
+================================
+
+Sunkit-pyvista and can also be used to plot field lines from `pfsspy`.
+"""
+
+import numpy as np
+from pfsspy.sample_data import get_gong_map
+
+import astropy.units as u
+from sunpy.data.sample import AIA_193_IMAGE
+from sunpy.map import Map
+
+from sunkit_pyvista import SunpyPlotter
+
+###############################################################################
+# We will firstly use an AIA 193 image from the sunpy sample data as the base image.
```

Comment:
This will be an issue if the AIA map and the GONG map were not taken at the same time. I will see if I can add a GONG map to the pfsspy sample data that is taken at the same observation date as the AIA map. Lets not block this PR on this point, but we should fix it before releasing the package!

Reply:
Oh okay, yes